### PR TITLE
Feature: Make self-issued tokens one-time-use only

### PR DESCRIPTION
**Pull request by aleksandra-bel**
This is implementation of Make self-issued tokens one-time-use only
Closes: https://github.com/eclipse-tractusx/managed-identity-wallet/issues/264

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: src/main/resources/db/changelog/changes/create_jti_table.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ * ******************************************************************************
+ */
+
```
Comment (borisrizov-zf) [inline src/main/resources/db/changelog/changes/create_jti_table.sql:21]: Add the liquibase required comments:
```
-- liquibase formatted sql
-- changeset andreibogus:create-jti-table
```

**Review by borisrizov-zf — CHANGES_REQUESTED** [review]
Looks good. Just add the liquibase comments and test it.

File: src/main/resources/db/changelog/changes/create_jti_table.sql (hunk 1 above)
Comment (andreibogus) [reply src/main/resources/db/changelog/changes/create_jti_table.sql:21]: Done in [9b277d7](https://github.com/eclipse-tractusx/managed-identity-wallet/pull/268/commits/9b277d7fe00aab4ab215eb3f91896d3429daffae)

**Review by borisrizov-zf — APPROVED** [review]
I'll test locally and merge afterwards. Look good!

**Review by borisrizov-zf — APPROVED** [review]
Doing local tests, and will merge afterwards.

**Review by borisrizov-zf — CHANGES_REQUESTED** [review]
Found a bug when re-wrapping an access token.

**Comment by andreibogus** [comment]
Fixes for re-wrapping an access token applied in [e047550](https://github.com/eclipse-tractusx/managed-identity-wallet/pull/268/commits/e04755016aecdf6d6ade0eebae065f36ce80d429)

**Review by borisrizov-zf — APPROVED** [review]
Looks good, I'll do the manual tests and ping you if anything arises.

**Pull request merged**